.mvn/rrf/prefixes-central.txt

```diff
@@ -0,0 +1,19246 @@
+## repository-prefixes/2.0
+/abbot
+/academy
+/acegisecurity
+/activation
+/activecluster
+/activeio
+/activemq
+/activemq-jaxb
+/activesoap
+/activespace
+/adarwin
+/ae
```

Comment:
i would expected "use Central by default" logic
ie there should be no need for this list.

Reply:
Unsure what you mean by "use central by default" (repo ordering maybe?) and why you think there is no need for this (can you elaborate what you mean here?), as this prevents Central to be asked for things known ahead of time that are not to be there (so prevents HTTP requests that are known to result in HTTP 404, but have to note that opposite is not true: if prefix is present in file, it does not mean that anything with that prefix is present in central either).

Reply:
Is it possible to do negative filtering? In other words, allow all prefixes except these?

Reply:
Currently no, as "prefixes" were originally invented as inverted flow: they are usually generated by remote (central, or some MRMs), and the remote essentially tells you which prefixes they have, so spare them with HTTP requests for non-matching prefixes.

Today there are two filter implementations (but is in resolver SPI, so is open game):
* prefixes (contained prefixes by $REPO) - usually provided by remote
* groupIds (allowed groupIds from $REPO) - usually authored by project (local)

With combining these two, most problems can be solved, see https://github.com/cstamas/rrf-demo demo where a "bad project doing all wrong" is fixed by adding RRF around it only, and no other change (this is just an example, in real world the bad project would need to be fixed as well).